src/components/TodoComponents/TodoForm.js

```diff
@@ -0,0 +1,54 @@
+import React, { Component } from 'react'
+
+export default class Drawer extends Component {
+    constructor(props) {
+        super(props)
+        
+        this.state={
+            name:''
+        }

```

Comment:
The goal of this application was to centralize state in your main app component and pass down anything the children components need through prop drilling.